package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@babel/core": "7.1.0",
     "@pdftron/webviewer": "^7.1.0",
-    "@pdftron/webviewer-video": "^3.0.3",
+    "@pdftron/webviewer-video": "file:../WebViewer-video",
```

Comment:
While testing, please change this to your own local folder of WebViewer-video in the branch `feature/load-from-local` from [this PR](https://github.com/XodoDocs/WebViewer-video/pull/22).